models/models.js

```diff
@@ -82,3 +82,33 @@ exports.insertComments = (id, newComment) => {
     return result.rows[0];
   });
 };
+
+exports.updateReview = (id, voteAmount) => {
+  let queryString = `
+        UPDATE reviews
+        SET votes = votes + ${voteAmount}
+        WHERE review_id = ${id}
```

Comment:
Template literals leaves you open to SQL injection, remember paprmetric queries!